docs/infrastructure/infra-ui.asciidoc

```diff
@@ -2,67 +2,112 @@
 [[infra-ui]]
 == Using the Infrastructure UI
 
-Use the Infrastructure UI in {kib} to monitor your infrastructure and identify
-problems in real time. 
+// +++ Gail provides this as an example of a good document. Leave it as is!
 
-You start with an overview of your infrastructure.
-Then you can use the interactive UI to drill down into areas of interest.
+Use the Infrastructure UI in {kib} to monitor your infrastructure and identify problems in real time.
+You can explore metrics for hosts, containers and services.
+You can also drill down to view more detailed metrics, or seamlessly switch to view the corresponding logs, application traces and uptime information.
 
-[role="screenshot"]
-image::infrastructure/images/infra-sysmon.jpg[Infrastructure Overview in Kibana]
+Initially, the *Inventory* tab shows an overview of the hosts in of your infrastructure and the current CPU usage for each host.
+From here, you can drill down into areas of interest.
 
-TIP: Right-click an element to jump to related logs or metrics. 
+[role="screenshot"]
+image::infrastructure/images/infra-sysmon.png[Infrastructure Overview in Kibana]
 
 [float]
 [[infra-cat]]
-=== View your infrastructure by hosts or container
+=== Choose the high-level view of your infrastructure
 
-Select the high-level view: *Hosts*, *Kubernetes*, or *Docker*.
-When you change views, you can see the same data through the perspective of a
-different category.
+Select the high-level view from *Hosts*, *Kubernetes*, or *Docker*.
+When you change views, you see the same data through the perspective of a different category.
 
-[float]
-[[infra-search]]
-=== Use the power of Search
-
-The Search bar is always available. You can use it to perform adhoc or structured searches.
-The Search feature can surface the information you're looking for, even if the
-other options you have selected would normally filter it out.
+The default representation is the *Map view*, which shows your components in a _waffle map_ of one or more rectangular grids.
+If the view you select has a large number of components, you can hover over a component to see the the details for that component. Alternatively, if you would prefer to see your infrastructure as a table, click *Table view*.
 
 [float]
 [[infra-metric]]
-=== Start at a high-level by selecting the metric
+=== Select the metric to view
 
-This filter helps you start focusing on potential problem areas that may need
-further investigation. You'll see metrics that are most relevant for hosts or
-the container you selected.
+Select the metric to view from the *Metric* dropdown list.
+The available metrics are those that are most relevant for the high level view you selected.
```

Comment:
```suggestion
The available metrics are those that are most relevant for the high-level view you selected.
```
Maybe?

Reply:
Here, I think we have a compound modifier before a noun, so the hyphen is needed, as you suggest, according to https://developers.google.com/style/hyphens. So in goes the hyphen